### ISSUE TITLE
Combobox: Add missing `a11y` tests

### ISSUE THREAD

**Issue by jordanjones243** [AlaskaAirlines/auro-formkit#1320]
### Description

Add the `auro-library` `useAccessibleIt()` tool to the combobox test file to ensure there is stable accessibility across all states of combobox

### Additional context

At the current state, adding this function will present test failures. There will need to be refactoring of tests and/or code to get these a11y tests to pass.

### Exit criteria (To be determined by the Auro team)

All tests pass using the `useAccessibleIt` tool
